api/src/main/java/com/example/api/error/exception/ExceptionMessage.java

```diff
@@ -21,4 +21,5 @@ public class ExceptionMessage {
     public static final String OPTION_FORM_FIELDS ="Każde pole w odpowiedzi do pytania powinno miec wartość";
     public static final String INVALID_QUESTION_TYPE ="Pole questionType powinno mieć wartość OPENED / SINGLE_CHOICE / MULTIPLE_CHOICE";
     public static final String INVALID_DIFFICULTY ="Pole difficulty powinno mieć wartość EASY / MEDIUM / HARD";
+    public static final String CONTENT_LEN_TO_BIG ="Liczba znaków w polu content nie może przekraczać ";
```

Comment:
CONTENT_LEN_TOO_BIG ja bym może dał, wiem że to pierdoła, ale rzuciło mi się to w oczy

Reply:
racja, angielski [*]